collection/alok.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "alok",
+    "meaning": "Brightness",
+    "aliases": [
+        "alok"
+    ],
+    "translations": {
+        "heb": "אלוק",
+        "rus": "Alok"
```

Comment:
Hi @alokraj68 
Please make sure the translation is written with the correct alphabet. Also, all of the translations should start with a lower case letter.
Will gladly merge once changes are made, thanks!